app/controllers/repair_requests_controller.rb

```diff
@@ -20,11 +20,15 @@ def new
         sor_code: @keyfax_result&.sor_code,
         }],
       priority: @keyfax_result&.priority,
-      description: @keyfax_result&.fault_text
+      description: @problem_description
     )
 
     @cautionary_contacts = Hackney::CautionaryContact.find_by_property_reference(@property.reference)
     @keyfax_session = Hackney::KeyfaxSession.create(current_page_url: new_property_repair_request_url(@property.reference))
+
+    cautionary_contact_alert_codes = get_cautionary_contact_alert_codes(@cautionary_contacts)
+
+    @problem_description = "#{cautionary_contact_alert_codes} #{@keyfax_result&.fault_text}"
```

Comment:
If you move this literal to line 23, you can get rid of the `@problem_description` variable and the `value: ` on the view, as it will be picked up automatically by the form helpers.